src/common/Main/styled.js

```diff
@@ -1,10 +1,12 @@
 import styled from "styled-components";
 
 export const MainContainer = styled.main`
-  max-width: 1368px;
+  max-width: 1408px;
   margin: 56px auto 40px auto;
+  padding: 0 20px;
 
   @media (max-width: ${({ theme }) => theme.breakpoints.mobileMax}) {
     margin: 24px auto 32px auto;
+    padding: 0 10px;
```

Comment:
Dla mnie zmiana na mały plus, bo w sumie nie ma dużo takich rozdzielczości, że te komponenty stykają się z ekranem, ale jednak są i to zabezpieczenie jest dobre 👍

Reply:
Jest bardzo dużo takich rozdzielczości, wszystkie poniżej 1368px 🙈